pkg/module/prepare_env.go

```diff
@@ -0,0 +1,67 @@
+package module
+
+import (
+	"fmt"
+	"os"
+	"path/filepath"
+	"runtime"
+)
+
+// prepareEnv will return all the appropriate
+// environment variables for a Go Command to run
+// successfully (such as GOPATH, GOCACHE, PATH etc)
+func prepareEnv(gopath, goProxy string) []string {
```

Comment:
I'm not strictly against it but does moving this function to a separate file improves anything? Having to jump to another file to read it seems cumbersome to me when the function is used only in one place.

Reply:
it is used by `go_get_fetcher` and `go_vcs_lister` both. 
May we merge lister and fetcher, then all the three could be put in one file.
But for now, they implement different interface, i think it would be better to keep them independent to make the refactor clear.